src/methods.jl

```diff
@@ -38,3 +38,38 @@ julia> bias_subtraction(frame, bias)
 * [`bias_subtraction!`](@ref)
 """
 bias_subtraction(frame::AbstractArray, bias_frame::AbstractArray) = bias_subtraction!(deepcopy(frame), bias_frame)
+
+
+"""
+    overscan_subtraction(frame::AbstractArray, overscan_frame::AbstractArray; overscan_axis = 2)
+
+In place version of [`overscan_subtraction`](@ref)
+"""
+function overscan_subtraction!(frame::AbstractArray, overscan_frame::AbstractArray; overscan_axis = 2)
+    combined_overscan = median(overscan_frame, dims = overscan_axis)
+    frame .-= combined_overscan
+    return frame
+end
```

Comment:
in general, you should use `dims` instead of `overscan_axis` since that is much more consistent with the expected behavior and Julia functions. 

I would be careful about setting the default to `dims=2`, though. Modern CCDs read out row-by-row, so it makes sense to choose some dimension, but it's tough to assume that `dims=2` is row-wise or col-wise in terms of the CCD geometry. 

For now, I would set `dims=Colon()` as a default but we should revisit this since it has statistical implications.

Reply:
Sure!